src/Frank/OpenApi.fs

```diff
@@ -0,0 +1,62 @@
+module Frank.OpenApi
+
+open System.Collections.Generic
+open System.Threading.Tasks
+open Microsoft.AspNetCore.Builder
+open Microsoft.AspNetCore.Http
+open Microsoft.AspNetCore.Routing
+open Microsoft.AspNetCore.Routing.Template
+open Microsoft.Extensions.DependencyInjection
+open Microsoft.OpenApi
+open Microsoft.OpenApi.Extensions
+open Microsoft.OpenApi.Models
+
+module OperationType =
+
+    let ofHttpMethod httpMethod =
+        if HttpMethods.IsGet httpMethod then OperationType.Get
+        elif HttpMethods.IsPut httpMethod then OperationType.Put
+        elif HttpMethods.IsPost httpMethod then OperationType.Post
+        elif HttpMethods.IsDelete httpMethod then OperationType.Delete
+        elif HttpMethods.IsOptions httpMethod then OperationType.Options
+        elif HttpMethods.IsHead httpMethod then OperationType.Head
+        elif HttpMethods.IsPatch httpMethod then OperationType.Patch
+        elif HttpMethods.IsTrace httpMethod then OperationType.Trace
+        else failwithf "Unrecognized Open API HTTP method %s." httpMethod
```

Comment:
Could do a literal comparison here, but I thought this would be more flexible in the event the `HttpMethods` were not traditional `GET`, `PUT`, etc.